app/assets/stylesheets/components/_navigation.scss

```diff
@@ -1,10 +1,10 @@
-.rad__authentication {
+.rad__navigation {
```

Comment:
Ah yes, good point. It was initially due to a naming clash with frontend's `.authentication`, but now we've renamed it we don't have to worry about that anymore.